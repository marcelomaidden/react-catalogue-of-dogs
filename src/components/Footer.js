import React from 'react';
import '../stylesheets/Footer.css';

const Footer = () => (
  <footer className="p-5">&copy; 2021 - Marcelo Fernandes de Araujo</footer>
);

export default Footer;
