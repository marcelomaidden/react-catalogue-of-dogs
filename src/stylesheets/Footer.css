footer {
  background-image: linear-gradient(to right, rgba(51, 51, 51, 0.8) 75%, rgba(51, 51, 51, 0.3));
}