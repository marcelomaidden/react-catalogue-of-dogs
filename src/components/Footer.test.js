import React from 'react';
import { render, unmountComponentAtNode } from 'react-dom';
import { act } from 'react-dom/test-utils';

import Footer from './Footer';

let container = null;
beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it('should render a Footer component', () => {
  act(() => {
    render(
      <Footer />,
      container,
    );
  });

  expect(container.innerHTML).toMatchSnapshot();
});
