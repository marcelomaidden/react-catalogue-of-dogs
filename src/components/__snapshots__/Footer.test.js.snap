// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Footer component 1`] = `"<footer class=\\"p-5\\">Â© 2021 - Marcelo Fernandes de Araujo</footer>"`;
